hazelcast/src/main/java/com/hazelcast/internal/partition/operation/PartitionBackupReplicaAntiEntropyOperation.java

```diff
@@ -79,7 +79,7 @@ public void run() {
         }
 
         Address ownerAddress = partition.getOwnerOrNull();
-        if (!getCallerAddress().equals(ownerAddress)) {
+        if (ownerAddress != null && !getCallerAddress().equals(ownerAddress)) {
```

Comment:
I don't think this `null` guard is needed any more. It was introduced because `ownerAddress` would cause NPE  when used like this:
```
getAllKnownAliases(getCallerAddress()).stream().noneMatch(ownerAddress::equals)
```

But `getCallerAddress().equals(ownerAddress)` will work just fine if `ownerAddress` is `null`.

Reply:
Done.